From 84ed3a0bee5f89f2ee5f84a07fb5de4ed5d84c92 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Sun, 3 Nov 2019 00:07:26 +0100
Subject: [PATCH] lib/goom2k4-0: fix static build

Fixes build error:

[100%] Linking CXX shared library visualization.goom.so
/home/buildroot/br8/output/host/opt/ext-toolchain/bin/../lib/gcc/x86_64-buildroot-linux-uclibc/7.4.0/../../../../x86_64-buildroot-linux-uclibc/bin/ld:
 lib/goom2k4-0/libgoom.a(goom_core.c.o):
 relocation R_X86_64_32S against `.rodata' can not be used when making
 a shared object; recompile with -fPIC

Patch sent upstream as PR 19.

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 lib/goom2k4-0/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/goom2k4-0/CMakeLists.txt b/lib/goom2k4-0/CMakeLists.txt
index e9b1fc7..304805e 100644
--- a/lib/goom2k4-0/CMakeLists.txt
+++ b/lib/goom2k4-0/CMakeLists.txt
@@ -51,3 +51,4 @@ set(GOOM_HEADERS src/goom.h
                  src/xmmx.h)
 
 add_library(goom STATIC ${GOOM_SOURCES} ${GOOM_HEADERS})
+set_property(TARGET goom PROPERTY POSITION_INDEPENDENT_CODE ON)
-- 
2.20.1

